libbeat/common/transport/tlscommon/verify.go

```diff
@@ -69,7 +74,7 @@ func verifyCertificateExceptServerName(
 	for i, asn1Data := range rawCerts {
 		cert, err := x509.ParseCertificate(asn1Data)
 		if err != nil {
-			return nil, nil, errors.Wrap(err, "tls: failed to parse certificate from server")
+			return nil, nil, fmt.Errorf("tls: failed to parse certificate from server: %+v", err)
```

Comment:
It must say:

```
fmt.Errorf("tls: failed to parse certificate from server: %+w", err)
```

Using `%w` makes Errorf expect an error value that will be wrapped, making the resulting error value equivalent to errors.Wrap. The inner error can be queried via `Unpack`.